### PR TITLE
ath79: add support for Ubiquiti UniFi AC-Pro

### PR DESCRIPTION
**Pull request by lucize**
rework the dts to a common unifi-ac dtsi
pro network is connected via phy0 and has usb ports
lite network is connected via phy4 without usb ports

everything works

Signed-off-by: Lucian Cristian <lucian.cristian@gmail.com>

### CODE REVIEW

**Comment by juppin** [comment]
You could add `ATH_SOC := qca9563` and `DEVICE_PACKAGES += kmod-ath10k ath10k-firmware-qca988x` to `Device/ubnt-unifiac` and remove this from the device image targets.
And probably you should explain more what are you doing in with the commit message...

**Comment by lucize** [comment]
yes, I should add more details to commit, I'll rework the image creation

**Comment by lucize** [comment]
@juppin should be better now

File: target/linux/ath79/image/generic-ubnt.mk [hunk 1]
```diff
@@ -81,20 +81,25 @@ endef
...
   IMAGES := sysupgrade.bin
-  IMAGE/sysupgrade.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | check-size $$$$(IMAGE_SIZE)
+  IMAGE/sysupgrade.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
 endef
 
 
 define Device/ubnt-unifiac-lite
   $(Device/ubnt-unifiac)
   DEVICE_TITLE := Ubiquiti UniFi AC-Lite
   DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca988x
```
Comment (juppin) [inline target/linux/ath79/image/generic-ubnt.mk]: Why not remove this and do a `DEVICE_PACKAGES += kmod-usb-core kmod-usb2` on Device/ubnt-unifiac-pro

Comment (lucize) [reply target/linux/ath79/image/generic-ubnt.mk]: yeah, I guess is better that way

**Comment by mkresin** [comment]
Thanks! Merged to [my staging tree](https://git.openwrt.org/?p=openwrt/staging/mkresin.git;a=summary).

I've applied some more cosmetic changes. Would you please test an image build from my staging tree! I would like to ensure that I haven't broke anything.

**Comment by lucize** [comment]
@mkresin I've built a clean image from your staging and all is working

**Pull request closed without merging**